src/console.cpp

```diff
@@ -26,7 +26,10 @@ This file is part of DarkStar-server source code.
 #include <shobjidl.h>
 
 /* Global Externs */
-extern bool g_Hide;
+namespace globals
+{
+    extern bool g_Hide;
```

Comment:
I'll do a pass at this eventually to get rid of all extern stuff, but out of scope for now 👍